components/project-card/meta/title.jsx

```diff
@@ -24,7 +24,7 @@ class Title extends React.Component {
       );
     }
 
-    return <h4 className={classnames}>{title}</h4>;
+    return <h4 id="entry-title" className={classnames} aria-level={this.props.ariaLevel}>{title}</h4>;
```

Comment:
I completely get where you're coming from with the `aria-level` attr. here & I'm going to update that. Butttttt, as for the `id`, I can rename it to be something more specific. However, I'd prefer to keep this because in this instance, it's not about using it for targeted styles or js, but for the attr. `aria-labelledby`.

Because of ui layout, the html structure doesn't include the title in the same container as the primary content, so the title isn't technically associated with the actual content (_in this case `<article>`_). So, I used `aria-labelledby` on `<article>` [here](https://github.com/mozilla/network-pulse/pull/1130/files/924e4bef9d36cf3c8559fb3942dc599abd20099c#diff-9ce64733fe7a204e270fad8337eecef5R249) that corresponds with it's title/heading (using this `id`) to manually create that relationship. 

I used this [ref](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) to guide implementation.